packages/paste-core/components/sidebar/src/SidebarCollapseButton.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Button} from '@twilio-paste/button';
+import type {ButtonProps} from '@twilio-paste/button';
+import type {BoxProps} from '@twilio-paste/box';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {ChevronDoubleLeftIcon} from '@twilio-paste/icons/esm/ChevronDoubleLeftIcon';
+import {ChevronDoubleRightIcon} from '@twilio-paste/icons/esm/ChevronDoubleRightIcon';
+
+import {SidebarContext} from './SidebarContext';
+
+export interface SidebarCollapseButtonProps extends React.HTMLAttributes<HTMLButtonElement>, Pick<BoxProps, 'element'> {
+  i18nCollapseLabel: string;
+  i18nExpandLabel: string;
```

Comment:
suggestion: can these have defaults so that they don't have to be required?

Reply:
I think it's better to make them required as to avoid oversight. It's a small price to pay